src/pages/AdminManageLC.jsx

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 import { useState, useEffect } from 'react';
 import { Institution } from '../libs/ajax/Institution';
-import { User } from '../libs/ajax/User';
 import { LibraryCard } from '../libs/ajax/LibraryCard';
-import { Notifications, USER_ROLES } from '../libs/utils';
+import { Notifications } from '../libs/utils';
```

Comment:
Minor request, we should update imports to refer to `src` when possible.
```suggestion
import { Notifications } from 'src/libs/utils';
```